app/(tabs)/profile.tsx

```
@@ -8,13 +8,15 @@ import {
   ActivityIndicator,
   Alert,
 } from 'react-native';
+import ConfirmModal from '../../src/components/common/ConfirmModal';
```

ESLint

[error] `../../src/components/common/ConfirmModal` import should occur after import of `react-i18next`

(import/order)